README.md

```diff
@@ -1,2 +1,47 @@
-# new-website
-New IATI (about &amp; guidance) website based on Django and Wagtail CMS
+# IATI Website (pre-Alpha)
+This repository hosts the new IATI website based on Django and Wagtail CMS.  A PostgreSQL database stores the underlaying content and user data.
+
+The current scope of the project (to April 2018) focuses on the 'About IATI' and 'Publisher guidance' sections.
+
+
+## Pre-requite packages
+
+- Python3
+- PostgreSQL
+
+
+## Dev setup
+```
+# Set-up and activate virtual environment
+python3 -m venv pyenv
+source pyenv/bin/activate
+
+# Enter into project directory
+cd iati
+
+# Install requirements
+pip install -r requirements-dev.txt
```

Comment:
We need to think about making this consistent throughout our code base. Suggest we stick to requirements_dev.txt.

Reply:
pyIATI uses `requirements-dev.txt`, but most other repos have `requirements_dev.txt`.

Reply:
25dd37f

Reply:
We really need to fix consistency on this one. Suggest adding to monday projects to look at.